src/Phoenix.vue

```diff
@@ -77,7 +95,6 @@ export default {
   }
   .sidebar { grid-area: sidebar;
             background-color: brown;
-            width: 300px;
```

Comment:
I need this width for another project :man_facepalming:

Reply:
we should adjust the value other ways, then setting it to fixed 300px - not sure what the exact requirement looks like - i'll check on that

Reply:
well - the requirment is; I need at least ~ 280px on the side bar to display all necessary information properly :man_shrugging:

Reply:
now i get your point - is this a requirement for EVERY device -screensize

Reply:
as of now - yes - in the future on a mobile screen we need a concept where the sidebar consumes the full screen ..... later ...